ts-graphql/src/index.ts

```diff
@@ -26,16 +26,16 @@ async function bootstrap() {
   server.applyMiddleware({ app });
 
 
-  if (process.env["CLOUDCC"] != "true") {
-    app.listen(4000, async () => {
-      console.log(`Server is running on http://localhost:4000/graphql`)
-    })
-  }
+  /**
+   * @capability https_server
+   * 
+   * @klotho::public
```

Comment:
what's this @klotho for?

Reply:
I think it's a no-op right now, but I believe the intent Gordon had is to signal that this is a Klotho annotation, and preparing for the V1 annotation changes

Reply:
I think it's a no-op right now, but I believe the intent Gordon had is to signal that this is a Klotho annotation, and preparing for the V1 annotation changes